src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -50,7 +51,9 @@ public virtual void OnRequestShowFileDialog(object sender, PackagePathEventArgs
         }
 
         private readonly IPreferences setting;
```

Comment:
Can you convert this setting to a property to return the value from LoadedPackageParams.